lib/libzpool/util.c

```diff
@@ -149,16 +149,14 @@ show_pool_stats(spa_t *spa)
 	nvlist_free(config);
 }
 
-/* *k_out must be freed by the caller */
 static int
 set_global_var_parse_kv(const char *arg, char **k_out, u_longlong_t *v_out)
 {
 	int err;
 	VERIFY(arg);
-	char *d = strdup(arg);
 
 	char *save = NULL;
-	char *k = strtok_r(d, "=", &save);
+	char *k = strtok_r((char *)arg, "=", &save);
```

Comment:
You are explicitly dropping const qualifier on a function that is known to modify the string.  How is that correct?

Reply:
I thought that this function did not modify the string. I will check the c library implementation and get back to you.

Reply:
The man page says that it modifies the string. The musl implementation does not while the glibc implementation does. Anyway, I am convinced that I was mistaken to think that there were no changes. I am fixing this in another way. I will repush the new version.

Thanks for catching this.